frontend/src/metabase/services.js

```diff
@@ -59,7 +60,7 @@ export function maybeUsePivotEndpoint(api, card, metadata) {
   }
   if (
     question.display() !== "pivot" ||
-    !question.isStructured() ||
+    !isStructured(card) ||
```

Comment:
Why aren't we using `question.query()` here?

Reply:
Because this endpoint is used in Audit internal dashboards, and it explodes when you try to construct MLv2 query with the `internal` query.

Reply:
I can add a comment, if you think it's necessary?

Reply:
I don't think it's necessary. Thanks!